[#15] Add spinners

YU-K opened the pull request:
Added a spinner display
when adding a new todo list and when deleting it.
Also when adding,
deleting and completing a task

YU-K committed:
[#15] Add spinners

Added a spinner display
when adding a new todo list and when deleting it.
Also when adding,
deleting and completing a task

AlexandrKoliukh commented on src/features/lists/NewListForm.jsx:
```diff
@@ -45,33 +46,36 @@ const NewListForm = () => {
+              )}
+            </div>
+          </Form>
+          {isSubmitting ? <Spinner /> : null}
```
Тут неоч, можно из бутстрапа взять https://getbootstrap.com/docs/4.4/components/spinners/#buttons

AlexandrKoliukh commented on src/features/lists/List.jsx:
```diff
@@ -54,21 +55,29 @@ const List = ({ list }) => {
     'btn-link'
   );
 
+  const renderList = () => {
```
Нет смысла выносить в константу внутри компонента, так только читать сложнее. Можно сделать хок для спинера 

```
<Spin spinning={true}>
  <>
    <Component/>
    <Component/>
  </>
</Spin>
```

AlexandrKoliukh commented:
@YU-K

YU-K committed:
[#15] Add spinners #2

Change spinner size. Remove renderList, renderTask functions, add
spinners inside a buttons

YU-K committed:
[#15] Add spinners #2

Change spinner size. Remove renderList, renderTask functions, add
spinners inside a buttons

YU-K committed:
Merge branch 'loaders' of https://github.com/YU-K/react-todo-app-with-backend into loaders

YU-K committed:
Merge branch 'AlexandrKoliukh:master' into loaders
